src/connection/connection_defs.h

```diff
@@ -151,7 +152,8 @@ enum css_server_request
   SERVER_DEREGISTER_HA_PROCESS = 12,
   SERVER_GET_EOF = 13,
   SERVER_RECEIVE_MASTER_HOSTNAME = 14,
-  SERVER_CONNECT_NEW_SLAVE = 15
+  SERVER_CONNECT_NEW_SLAVE = 15,
+  SERVER_CONNECT_NEW_SLAVE_CONTROL_CHANNEL = 16
```

Comment:
We have to explore issues regarding "master incompatibility". I'd like to propose to go this way and open an issue to follow the issues.